auditlog/views/auditlog_view.xml

```diff
@@ -257,4 +257,99 @@
+                <field name="log_type" optional="hide" />
+            </tree>
+        </field>
+    </record>
+    <record id="view_auditlog_line_search" model="ir.ui.view">
+        <field name="name">auditlog.line.search</field>
+        <field name="model">auditlog.log.line</field>
+        <field name="arch" type="xml">
+            <search string="Logs Report">
+                <field name="name" />
+                <field name="model_id" />
+                <field name="res_id" />
+                <field name="user_id" />
+                <group expand="0" string="Group By...">
+                    <filter
+                        name="group_by_user_id"
+                        string="User"
+                        domain="[]"
+                        context="{'group_by':'user_id'}"
+                    />
+                    <filter
+                        name="group_by_model_id"
+                        string="Model"
+                        domain="[]"
+                        context="{'group_by':'model_id'}"
+                    />
+                    <filter
+                        name="group_by_field_id"
+                        string="Field"
+                        domain="[]"
+                        context="{'group_by':'field_id'}"
+                    />
+                    <filter
+                        name="group_by_res_id"
+                        string="Resource ID"
+                        domain="[]"
+                        context="{'group_by':'res_id'}"
+                    />
+                    <filter
+                        name="group_by_create_date"
+                        string="Date"
+                        domain="[]"
+                        context="{'group_by':'create_date'}"
+                    />
+                    <filter
+                        name="group_by_http_session"
+                        string="User session"
+                        domain="[]"
+                        context="{'group_by':'http_session_id'}"
+                    />
+                    <filter
+                        name="group_by_http_request"
+                        string="HTTP Request"
+                        domain="[]"
+                        context="{'group_by':'http_request_id'}"
+                    />
+                </group>
+            </search>
+        </field>
+    </record>
+    <record id="action_auditlog_line" model="ir.actions.act_window">
+        <field name="name">Auditlog Report</field>
+        <field name="res_model">auditlog.log.line</field>
+        <field name="view_mode">tree</field>
+        <field name="search_view_id" ref="view_auditlog_line_search" />
+        <field name="context">{'search_default_group_by_model_id': 1}</field>
+    </record>
+    <menuitem
+        id="menu_auditlog_line"
+        name="Logs Report"
+        parent="menu_audit"
+        action="action_auditlog_line"
+        sequence="20"
```

Comment:
This moves the menu item at the bottom inside 'Audit'.  Could we perhaps add the sequences 30 and 40 to 'User sessions' and 'HTTP Requests', which currently do not seem to have a sequence specified?

![image](https://user-images.githubusercontent.com/7766939/130240394-ecc8593f-b7b3-4d8b-867b-af222d66e40c.png)

Reply:
+1

Reply:
Done